java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -292,13 +292,15 @@ public interface StringBaseCommands {
      * @see <a href="https://valkey.io/commands/mset/">valkey.io</a> for details.
      * @param keyValueMap A key-value map consisting of keys and their respective values to set.
      * @return Always <code>OK</code>.
+     * @param ArgType you can only pass Keys and Values of types {@link String} and {@link
+     *     GlideString} Key and Value must be of same type.
      * @example
      *     <pre>{@code
      * String result = client.mset(Map.of("key1", "value1", "key2", "value2"}).get();
      * assert result.equals("OK"));
      * }</pre>
      */
-    CompletableFuture<String> mset(Map<String, String> keyValueMap);
+    <ArgType> CompletableFuture<String> mset(Map<ArgType, ArgType> keyValueMap);
```

Comment:
Add comment what `ArgType` is in the doc. See [here for example](https://github.com/aws/glide-for-redis/blob/ed6ca3e33359429ce1deb36a6be97ae905e6771e/java/client/src/main/java/glide/api/models/BaseTransaction.java#L321)

Reply:
done